candevstudio/docs/README.md

```diff
@@ -0,0 +1,24 @@
+# Introduction
```

Comment:
To be reworked, copy/paste of sdk-devkit that should be updated to better match CANDevStudio doc.